src/main/java/org/schemaspy/cli/CommandLineArguments.java

```diff
@@ -102,6 +102,15 @@ public class CommandLineArguments {
     )
     private boolean noImplied = false;
 
+    @Parameter(
+        names = {
+            "-no", "--no-orphans",
```

Comment:
Should we really have `-no` and `schemaspy.no`, if so, the latter needs to be added to CombinedDefaultsProvider.

Reply:
I'm fine either way. It could be handled in #1031 if we just expand it to include consistency for alla args and not just long ones. Since I think it's probably best to check both so they are in synch. For now it's mainly that it's incomplete and I'm fine going either way.